### PR TITLE
Small improvements to style transfer

### PR DESCRIPTION
**Pull request by msluszniak**
## Description
Small improvements in readability of Style Transfer. Please double check it, as I haven't run the whole style transfer.

### CODE REVIEW

File: ios/RnExecutorch/utils/ImageProcessor.m [hunk 1]
```diff
@@ -19,20 +19,22 @@ + (float *)imageToFloatArray:(UIImage *)image size:(CGSize *)outSize {
...
     CGContextRef context = CGBitmapContextCreate(rawData, width, height,
                                                  bitsPerComponent, bytesPerRow, colorSpace,
                                                  kCGImageAlphaPremultipliedLast | kCGBitmapByteOrder32Big);
 
     CGColorSpaceRelease(colorSpace);
     CGContextDrawImage(context, CGRectMake(0, 0, width, height), [image CGImage]);
 
-    float *floatArray = malloc(width * height * sizeof(float) * 3);
+    // only RGB, without alpha channel
+    float *floatArray = malloc(width * height * sizeof(float) * (bytesPerPixel - 1);
```
Comment (msluszniak) [inline ios/RnExecutorch/utils/ImageProcessor.m]:
```suggestion
    float *floatArray = malloc(width * height * sizeof(float) * (bytesPerPixel - 1));
```

File: ios/RnExecutorch/models/StyleTransferModel.mm [hunk 2]
```diff
@@ -49,10 +52,10 @@ - (UIImage *)postprocess:(UIImage *)input {
...
   UIImage *processedImage = [self preprocess:input];
-  CGSize outputSize = {640, 640};
+  CGSize outputSize = {width_size, height_size};
   float* processedImageData = [ImageProcessor imageToFloatArray:processedImage size:&outputSize];
   
-  NSArray *modelInput = [self floatArrayToNSArray:processedImageData length:1228800];
+  NSArray *modelInput = [self floatArrayToNSArray:processedImageData length:(width_size * height_size * num_channels)];
   NSNumber* numInputs = [module getNumberOfInputs];
   NSLog(@"RnExecutorch: %@", [module getNumberOfInputs]);
   for (NSUInteger i = 0; i < [[module getNumberOfInputs] intValue]; i++) {
```
Comment (chmjkb) [inline ios/RnExecutorch/models/StyleTransferModel.mm]: Can you also get rid of that for loops that only log stuff there? They don't seem necessary, thank you for contributing!:D

Comment (msluszniak) [reply ios/RnExecutorch/models/StyleTransferModel.mm]: Sure, I'm wondering if this PR is still needed as there is a PR with style transfer using OpenCV

File: ios/RnExecutorch/models/StyleTransferModel.mm [hunk 3]
```diff
@@ -49,37 +52,17 @@ - (UIImage *)postprocess:(UIImage *)input {
 
 - (UIImage *)runModel:(UIImage *)input {
   UIImage *processedImage = [self preprocess:input];
-  CGSize outputSize = {640, 640};
+  CGSize outputSize = {width_size, height_size};
   float* processedImageData = [ImageProcessor imageToFloatArray:processedImage size:&outputSize];
   
-  NSArray *modelInput = [self floatArrayToNSArray:processedImageData length:1228800];
+  NSArray *modelInput = [self floatArrayToNSArray:processedImageData length:(width_size * height_size * num_channels)];
   NSNumber* numInputs = [module getNumberOfInputs];
```
Comment (msluszniak) [inline ios/RnExecutorch/models/StyleTransferModel.mm]:
```suggestion
```

**Comment by mkopcins** [comment]
This is a very nice work with great attention to details. However I will close this PR, because those files were supposed to be changed from the start (perhaps shouldn't have been merged in the first place, they were only to ease further development). We can implement suggested changes in the refactored code for style transfer :)

**Pull request closed without merging**